src/SprykerSdk/Acp/Validator/Finder/FinderInterface.php

```diff
@@ -29,9 +29,9 @@ public function hasFiles(?string $path): bool;
     /**
      * @param string $path
      *
-     * @return \Symfony\Component\Finder\SplFileInfo
+     * @return \Symfony\Component\Finder\SplFileInfo|null
```

Comment:
Please revert this. get has to return a file or throw a meaningful exception. Only find methods allow null. The finder already has a `hasFile` method use that before you try to get a file.